README.md

```diff
@@ -239,6 +239,14 @@ Container images used in the tests are:
   - version 10.3: `registry.redhat.io/rhscl/mariadb-103-rhel7` (only if `ts.redhat.registry.enabled` is set)
 - MSSQL: `mcr.microsoft.com/mssql/rhel/server`
 
+### `sql-db/sql-reactive`
```

Comment:
Could be renamed to `sql-db/sql-reactive-panache` because there are several way to work with SQL in reactive way

Reply:
Good point, I renamed the module.